beta/sdk/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:buster-slim
+
+RUN set -eux; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends \
+		ca-certificates \
+		curl \
+		openssh-client \
+		unzip \
+	; \
+	rm -rf /var/lib/apt/lists/*
+
+COPY install_dart.sh /usr/local/bin
+RUN /usr/local/bin/install_dart.sh 2.13.0-211.6.beta beta linux x64 /usr/lib/dart
```

Comment:
https://github.com/docker-library/official-images#clarity :eyes:

Reply:
Good point! I'll be refactoring (or @athomas or someone else on the team will). Just wanted to get something working quickly, but we'll make sure to convert to `RUN` instructions either with this draft PR or a following one.

Reply:
Done!